### PR TITLE
cmake changes

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

**Comment by badone** [comment]
LGTM

**Comment by jcsp** [comment]
Checking that this fixes the issue for me locally now...

**Comment by tchaikov** [comment]
the changeset is being tested at http://pulpito.ceph.com/kchai-2016-06-29_03:52:56-rados-wip-cmake-debian---basic-smithi/ .

**Comment by jcsp** [comment]
First two commits are good (tested locally before the later ones were added to this PR)

**Comment by cbodley** [comment]
:+1:

File: src/compressor/CMakeLists.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ add_library(compressor_objs OBJECT ${compressor_srcs})
 
 ## compressor plugins
 
-set(compressor_plugin_dir ${CMAKE_INSTALL_PKGLIBDIR}/compressor)
+set(compressor_plugin_dir ${CMAKE_INSTALL_FULL_PKGLIBDIR}/compressor)
```
Comment (tchaikov) [inline src/compressor/CMakeLists.txt]: to be consistent, should use the relative path.

**Pull request merged**